### ISSUE TITLE
Is there a tutorial for maddpg.py

### ISSUE THREAD

**Issue by ConnLiu** [iffiX/machin#1]
Hi,
  this project is really awesome and the codes is well structured! 
I have run some codes in examples/tutorials, but can not find some about MARL algorithms such as maddpg.
Since you have already implemented the maddpg.py and test_maddpg.py, I am wondering could you implement a tutorial for maddpg.py too? (It would be better to implemented more MARL algorithms, such as COMA, QMIX, VDN)

**Comment by iffiX** [comment]
Yes, I will add a simple example for every framework tomorrow. Also I would like to know what's the most valuable / interesting MARL algorithm you consider to be, as I don't have to time to implement, test, and optimize performance for all these algorithms...
Preferably a performance comparison chart for COMA, QMIX, VDN?

**Comment by iffiX** [comment]
Example added https://github.com/iffiX/machin/commit/40d729e6b3079176ab77479eed4aad20e3ec0ec0
